CHANGES.md

```diff
@@ -1,3 +1,9 @@
+### Unreleased
+
+* [#67](https://github.com/square/rails-auth/pull/67)
+  Remove `ca_file`, `require_cert`, and `truststore` options to X509 middleware.
```

Comment:
we should probably add here that the X509 middleware will no longer attempt to verify a certificate chain.